packages/blocks/cloudhealth/src/index.ts

```diff
@@ -0,0 +1,29 @@
+import { createCustomApiCallAction } from '@openops/blocks-common';
+import { createBlock } from '@openops/blocks-framework';
+import { BlockCategory } from '@openops/shared';
+import { cloudhealthAuth } from './lib/auth';
+import { graphqlAction } from './lib/graphql-action';
+
+export const cloudhealth = createBlock({
+  displayName: 'CloudHealth',
+  auth: cloudhealthAuth,
+  minimumSupportedRelease: '0.20.0',
+  logoUrl: 'https://static.openops.com/blocks/cloudhealth.png',
+  authors: [],
+  categories: [BlockCategory.FINOPS],
+  actions: [
+    createCustomApiCallAction({
```

Comment:
as we discussed offline, should we add maybe a link to their docs?
https://apidocs.cloudhealthtech.com/

Reply:
<img width="488" alt="image" src="https://github.com/user-attachments/assets/f2ecf508-5e8b-46e4-9335-7b550e4e9de4" />